dvc/stage/run.py

```diff
@@ -70,32 +73,33 @@ def cmd_run(stage, *args, checkpoint_func=None, **kwargs):
         kwargs["shell"] = False
         executable = os.getenv("SHELL") or "/bin/sh"
         warn_if_fish(executable)
-        cmd = _nix_cmd(executable, stage.cmd)
 
     main_thread = isinstance(
         threading.current_thread(),
         threading._MainThread,  # pylint: disable=protected-access
     )
-    old_handler = None
-    p = None
+    for _cmd in cmd:
+        logger.info("\trunning %s", _cmd)
```

Comment:
Let's not add `running` on each command of the list as there's already a line for the whole stage.
Let's either print them all as a list before running or one-by-one:
```console
Running stage "build":
	- echo hello
	- echo world
```

But I can see that the output gets messy if the script output gets mixed.
We could think of removing indent and add a ">" before the script:
```console
Running stage "build":
> echo hello
hello
> echo world
world
```

Reply:
I like the suggestion with `>`, accepted ;)